pkg/service/rtcservice.go

```diff
@@ -229,15 +229,23 @@ func (s *RTCService) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// give it a few attempts to start session
 	var cr connectionResult
 	var initialResponse *livekit.SignalResponse
-	for i := 0; i < 3; i++ {
+	for i := 0; ; i++ {
```

Comment:
sorry if I caused the confusion.. I don't think we should perma loop here. clients have logic to try different data centers. if we never return, it could prevent the client fallback logic. for example, what if this pool has ran out of media nodes? in that case it would be better to time out sooner so the client goes and tries another region